Composer/plugins/localPublish/src/index.ts

```diff
@@ -289,14 +289,8 @@ class LocalPublisher implements PublishPlugin<PublishConfig> {
       configList.push(config.MicrosoftAppPassword);
     }
     if (config.luis) {
-      if (config.luis.authoringKey) {
-        configList.push('--luis:endpointKey');
-        configList.push(config.luis.authoringKey);
-      }
-      if (config.luis.authoringRegion) {
```

Comment:
I see this removed and it suggests that the `--luis:endpoint` is not necessary when the user supplies an authoringKey instead of endpointKey. Is this correct?

Reply:
endpoint is the URL base on luis region, it's not necessary because c# runtime will make it itself in `builder.UseLuisConfigAdapter().UseLuisSettings()` in Program.cs file (It's a feature of C# SDK).